### ISSUE TITLE
Error: The method 'detach' isn't defined for the class 'FocusScopeNode'.

### ISSUE THREAD

**Issue by veglinus** [AndreHaueisen/flushbar#47]
```
Compiler message:
file:///Users/elev/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/flushbar-1.4.0/lib/flushbar.dart:235:15
: Error: The method 'detach' isn't defined for the class 'FocusScopeNode'.
 - 'FocusScopeNode' is from 'package:flutter/src/widgets/focus_manager.dart'
 ('file:///Users/elev/Documents/flutter/packages/flutter/lib/src/widgets/focus_manager.dart').
Try correcting the name to the name of an existing method, or defining a method named 'detach'.
    focusNode.detach();
              ^^^^^^
```

Seems to be because of:
https://github.com/flutter/flutter/pull/31909

Similar issue:
https://github.com/flutter/flutter/issues/31784

**Comment by veglinus** [comment]
Copy of https://github.com/AndreHaueisen/flushbar/issues/44
Oops.
